sdk/computefleet/Azure.ResourceManager.ComputeFleet/api/Azure.ResourceManager.ComputeFleet.netstandard2.0.cs

```diff
@@ -1,5 +1,6 @@
 namespace Azure.ResourceManager.ComputeFleet
 {
+	//cspell:ignore Rdma
```

Comment:
The API file is auto-generated, should not edit it.

Reply:
ok, will remove it.

Reply:
ok